src/Commands/core/XhprofCommands.php

```diff
@@ -53,17 +50,17 @@ public function xhprofPost($result, CommandData $commandData)
      */
     public function xhprofInitialize(InputInterface $input, AnnotationData $annotationData)
     {
-        if (self::xhprofIsEnabled($input)) {
+        if (self::xhprofIsEnabled()) {
             $config = Drush::config()->get('xh');
             $flags = self::xhprofFlags($config);
-            \xhprof_enable($flags);
+            self::xhprofEnable($flags);
         }
     }
 
     public static function xhprofIsEnabled()
     {
         if (Drush::config()->get('xh.link')) {
-            if (!extension_loaded('xhprof') && !extension_loaded('tideways')) {
+            if (!extension_loaded('xhprof') && !extension_loaded('tideways_xhprof')) {
```

Comment:
Perhaps add docs at top of file about tideways. Also expand Exception message to mention new Tideways.